site/index.ts

```diff
@@ -171,6 +185,13 @@ domReady(() => {
             setDefaultEditor(e.detail.editorType);
         }
     );
+
+    place2?.addEventListener(
```

Comment:
This is fine as-is, but you could even just leave it out if you wanted. All this does is set the default editor on page load to the last mode you used, which isn't super important for our dual editing test.